target_mvd/settings.py

```diff
@@ -135,6 +135,13 @@
 
 USE_TZ = True
 
+# Rest Framework
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.TokenAuthentication',
```

Comment:
Do you make a migration after changing the settings?

Reply:
I just did. It added something up in the User model but nothing related to this. What do you mean? @nsantos16

Reply:
Ok, now I see the migration file, great job!